go/vt/vtgate/vindexes/lookup_hash.go

```diff
@@ -146,7 +144,6 @@ type LookupHashUnique struct {
 
 // NewLookupHashUnique creates a LookupHashUnique vindex.
 func NewLookupHashUnique(name string, m map[string]string) (Vindex, error) {
-	log.Warningf("LookupHashUnique index (%q) it's being deprecated. Please use LookupUnique", name)
```

Comment:
I assume removing these is intentional (which I support) but that seems orthogonal to these changes.

Reply:
Yeah. TThere was too much spam in tests.